dependencies.yaml

```diff
@@ -0,0 +1,107 @@
+# Dependency list for https://github.com/rapidsai/dependency-file-generator
+ files:
+   all:
+     output: conda
+     matrix:
+       cuda: ["11.5"]
+       arch: [x86_64]
+     includes:
+       - common_build
+       - cpp_build
+       - cudatoolkit
+       - doc
+       - python_build
+ channels:
+   - rapidsai
+   - rapidsai-nightly
+   - dask/label/dev
+   - conda-forge
+   - nvidia
+ dependencies:
+   cudatoolkit:
+     specific:
+       - output_types: conda
+         matrices:
+           - matrix:
+               cuda: "11.5"
+             packages:
+               - cudatoolkit=11.5
+           - matrix:
+               cuda: "11.4"
+             packages:
+               - cudatoolkit=11.4
+           - matrix:
+               cuda: "11.2"
+             packages:
+               - cudatoolkit=11.2
+           - matrix:
+               cuda: "11.0"
+             packages:
+               - cudatoolkit=11.0
+   common_build:
+     common:
+       - output_types: [conda]
+         packages:
+           - cmake>=3.23.1,!=3.25.0
+       - output_types: conda
+         packages:
+           - ninja
+           - pip
+   cpp_build:
+     common:
+       - output_types: conda
+         packages:
+           - nvcc_linux-64=11.5
+           - c-compiler
+           - cxx-compiler
+           - librmm=23.02.*
+           - libraft-headers=23.02.*
+           - libraft-distance=23.02.*
+           - libcugraphops=23.02.*
+           - clang=11.1.0
+           - clang-tools=11.1.0
+           - boost
```

Comment:
Where do we depend on `boost`?

Reply:
Looks like we used to depend on boost (circa version 0.6) but we may have eliminated that dependency.  I'm testing without it now.

Reply:
Removing boost